README.md

```diff
@@ -1,12 +1,36 @@
 # Datadog Agent release for BOSH
 
-* For all stemcells (Python source implementation)
+* For debian and rpm based stemcells (more to come)
 * Automatically defines tags based on deployments, names and jobs
 * Process, network, ntp and disk integrations by default
 * Monit processes are added automatically to process integration
 * You can define additional integrations
 
 
+# What this does
+
+This includes the rpm and debian releases in the package and unpacks them in the dd-agent directory.
+
+While a source install would be preferable, we're balancing a number of concerns.
+
+1. We want to ensure a consistent deployment among all customers.
+1. We want to ensure a quick deployment.
+
+Compiling Python takes a very long time (it took up to 30 minutes for Python alone on some machines we tested it on).
+We also saw some disparities on some machines. It can creates issues if it stomps on the system python (which is hard to avoid in some cases) and it sometimes doesn't work.
+
+So, our solution for this was to use our embedded python that we already have in our packages and unpack those packages (rather than installing them).
+
+
+# Versioning
+
+We version the bosh release with the following scheme:
+
+`major.minor.bugfix.cfbuild`
```

Comment:
as we discussed, we may want to use a `cfBuildMajor.cfBuildMinor.agentVersion`versioning scheme instead here (ie close to what we have for docker-dd-agent), it would be more consistent with the pattern adopted by other bosh releases